mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -2346,41 +2346,29 @@ private void MakeAttributes (XmlElement root, IEnumerable<string> attributes, Ty
 	}
 
 	public static string MakeAttributesValueString (object v, TypeReference valueType)
-	{
-		if (v == null)
-			return "null";
-			if (valueType.FullName == "System.Type") {
-				var vTypeRef = v as TypeReference;
-				if (vTypeRef != null)
-					return "typeof(" + NativeTypeManager.GetTranslatedName (vTypeRef) + ")"; // TODO: drop NS handling
-				else
-					return "typeof(" + v.ToString () + ")";
+		{
```

Comment:
What happened to the indentation here? `MakeAttributesValueString()` is at 1 tab stop, yet this `{` is at 2 tab stops, and the body is at 3 tab stops?

I think Xamarin Studio is mucking things up...